### DISCUSSION TITLE
LF APAC 开源布道者分享系列之二：如何将写作进行下去 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#28] [Announcements]
# LF APAC 开源布道者分享系列之二：如何将写作进行下去 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/how-to-write-article-for-advocate-os/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实, 还有一个问题, 类似: [Groklaw \- The Daemon, the GNU & the Penguin ~ by Dr\. Peter Salus](http://www.groklaw.net/staticpages/index.php?page=20051013231901859)

很多坚持写作的人, 都有一个特征:

- 几十年如一日, 固定在一个网站中写作
- 这个网站一般都是私人网站
- 那些坚持在免费服务平台中写的人, 都随着对应平台的丢失而消失
- ...

那么问题来了, 

- 20年前架一个私人网站成本有多少?
- 在没有 Markdown 的年代,想在网络中发表文字, 要用什么格式写?
- ...

所以, 现在真的是个幸福的时代, 有 M$ 撑腰, 任何人, 甚至于不用学习任何 git 指令,
就可以随时在 GitHub 中通过 pages 服务发布一个私人网站,
而且, 所有内容并不会随着任何平台的消失而消失, 自动保存在 git 仓库中, 随时可以迁移到另外一个发布平台中;

当然, 可惜的是, 私人域名本身还是要有一定知识以及成本的,
这又是普通人难以坚持的...

所以 , web 1.0 时代只是给公司发布信息的能力, 
web2.0 时代只是给私人不重要的事儿发布能力,
web 3.0 时代, 是否有私人可以长久发布思考结果的场景呢? 关键是成本足以低到失业人士也可以支撑的...
